apps/greencheck/importers/importer_csv.py

```diff
@@ -0,0 +1,64 @@
+import requests
+import logging
+import pandas as pd
+import ipdb
+import re
+import ipaddress
+
+from apps.greencheck.importers.importer_interface import BaseImporter, Importer
+
+from django.conf import settings
+
+logger = logging.getLogger(__name__)
+
+
+class CsvImporter(BaseImporter):
+    # def __init__(cls):
+    # cls.hosting_provider_id = settings.Csv_PROVIDER_ID
+
+    def fetch_data_from_source(cls):
+        # TODO: fetch data from website dropping
+        # For now: expect that a csv is saved somewhere and fetch in the following way:
+        raw_data = pd.read_csv(
+            "apps/greencheck/fixtures/test_dataset_csv.csv", header=None
+        )
+        return cls.parse_to_list(raw_data)
+
+    def parse_to_list(cls, raw_data):
+        try:
+            list_of_ips = []
+
+            cls.validate_csv_file(raw_data)
+
+            if len(raw_data.columns) == 1:
+                # ASN or network (ip with subnet)
+                list_of_ips = list(raw_data.iloc[:, 0])
+            # elif len(data.columns) == 2:
+            # TODO: Implement a way to convert two ip addresses to a network
+            # # Start and ending ip range
+            # start_ip = data.iloc[:,0]
+            # end_ip = data.iloc[:,1]
+            # Create network from these
```

Comment:
Is it necessary to create an IP Network? We store IPranges in the database, and while a network can resemble an IP range that has a start and end IP, they're not necessarily the same thing.

Reply:
At the moment, the saving interface function works with IP networks.
The reasoning behind this was so that we could easily have a list with a simple dimensions.
This posed a problem before, so I can this fundamental part around by allowing ip ranges.